refinery/ui/source/js/commons/ctrls/group-member-edit-modal-ctrl.js

```diff
@@ -0,0 +1,135 @@
+     * @desc  View method to close modals
+     * @memberOf refineryApp.GroupMemberEditModalCtrl. Expects modalInstance inscope
+      ***/
+    function close () {
+      vm.modalInstance.close(vm.alertType);
+    }
+
+     /**
+     * @name demote
+     * @desc  View method for demoted a member from manager
+     * @memberOf refineryApp.GroupMemberEditModalCtrl
+    **/
+    function demote () {
+      vm.isLoading = true;
+      groupMemberService.remove({
+        uuid: vm.resolve.config.activeGroup.manager_group_uuid,
+        userId: vm.member.user_id
+      }).$promise.then(
+        function () {
+          groupDataService.update();
+          vm.alertType = 'success';
+          vm.responseMessage = 'Successfully demoted member ' + vm.member.username;
+          $timeout(function () {
+            vm.isLoading = false;
+            vm.modalInstance.close(vm.alertType);
+          }, 1500);
+        }
+      ).catch(function () {
+        vm.alertType = 'danger';
+        vm.responseMessage = 'Error, could not demote member ' +
+          vm.member.username + '. Last member and manager can not leave';
+        vm.isLoading = false;
+      });
+    }
+
+     /**
+     * @name promote
+     * @desc  View method for promoting a member to manager
+     * @memberOf refineryApp.GroupMemberEditModalCtrl
+    **/
+    function promote () {
+      vm.isLoading = true;
+      groupMemberService.add({
+        uuid: vm.resolve.config.activeGroup.manager_group_uuid,
+        user_id: vm.member.user_id
+      }).$promise.then(
+        function () {
+          groupDataService.update();
+          vm.alertType = 'success';
+          vm.responseMessage = 'Successfully promoted member ' + vm.member.username;
+          vm.isLoading = false;
+          $timeout(function () {
+            vm.modalInstance.close(vm.alerType);
+          }, 1500);
+        }
+      ).catch(function () {
+        vm.alertType = 'danger';
+        vm.responseMessage = 'Error Could not promote member ' + vm.member.username;
+        vm.isLoading = false;
+      });
+    }
+
+     /**
+     * @name remove
+     * @desc  View method for removing a member from group
+     * @memberOf refineryApp.GroupMemberEditModalCtrl
+    **/
+    function remove () {
+      vm.isLoading = true;
+      groupMemberService.remove({
+        uuid: vm.resolve.config.activeGroup.manager_group_uuid,
+        userId: vm.member.user_id
+      }).$promise.then(
+        function () {
+          groupDataService.update();
+          vm.alertType = 'success';
+          vm.responseMessage = 'Successfully removed member' + vm.member.username;
+          vm.isLoading = false;
+          $timeout(function () {
+            vm.modalInstance.close(vm.alertType);
+          }, 1500);
+        }
+      ).catch(function () {
+        vm.alertType = 'danger';
+        vm.responseMessage = 'Error, could not remove member'
+          + vm.member.username + '. Last member and manager can not leave';
```

Comment:
Space after "member" here and 10 lines above. It looks like the same errors appear above...
Would it make sense to pull these out into a utility?